android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```

Comment:
Unrelated change?

Reply:
I was unable to build the app to confirm that the name updates did not break the build without this update. Newer versions of the JDK require an updated gradle.

Reply:
Figured it was something along those lines